src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -70,6 +70,8 @@ public class CommandTestUtil {
     public static final String VALID_TASK_DESCRIPTION = "Complete PRS";
     public static final String VALID_ASSIGNMENT_NAME = "Finding ELDRIC";
     public static final String INVALID_ASSIGNMENT_NAME = "Finding BOYD";
+    public static final String VALID_PROGRESS = "PENDING";
```

Comment:
Might want to standardise your naming conventions here.
Change `VALID_PROGRESS` to `VALID_PROGRESS_PENDING`
Since "Done" = `VALID_PROGRESS_DONE`